lib/index.ts

```diff
@@ -117,9 +118,11 @@ export async function inspect(
   }
 
   if (options.scanAllUnmanaged) {
```

Comment:
```suggestion
  if (options.scanAllUnmanaged || options.allProjects) {
```
These should be interchangeable, and in future we can deprecate scanAllUnmanaged.